src/sentry/static/sentry/app/components/activity/feed.jsx

```diff
@@ -40,7 +40,7 @@ const ActivityFeed = createReactClass({
     };
   },
 
-  componentWillMount() {
+  UNSAFE_componentWillMount() {
```

Comment:
Do we need a ticket to fix this later, since if I understand this correctly, this is deprecated?

Reply:
Yes, I've added one.